src/expressions.rst

```diff
@@ -425,49 +506,16 @@ A :t:`place expression` can be moved out of when it denotes
 After a :t:`place expression` is moved out, the memory location it represented
 is deinitialized and shall not be read from until reinitialized.
 
-:dp:`fls_cPEMHZtPkctX`
-An :t:`immutable place expression` is a :t:`place expression` whose memory
-location cannot be modified.
-
-:dp:`fls_ku38h562vfyl`
-A :t:`mutable place expression` is a :t:`place expression` whose memory location
-can be modified. The following :t:`[place expression]s` are :t:`[mutable place
-expression]s`:
-
-* :dp:`fls_bt50fltfqcvn`
-  An :t:`index expression` whose :t:`type` implements the
-  :std:`core::ops::IndexMut` :t:`trait`,
-
-* :dp:`fls_6b4rwkrc1ap6`
-  A :t:`dereference expression` whose :t:`type` is ``*mut T``,
-
-* :dp:`fls_s4bhrpykzmm7`
-  A :t:`dereference expression` of a :t:`field` or a :t:`variable` whose
-  :t:`type` is ``&mut T``,
-
-* :dp:`fls_1tq2o2huda9l`
-  A :t:`dereference expression` whose :t:`type` implements the
-  :std:`core::ops::DerefMut` :t:`trait`,
-
-* :dp:`fls_xm0gm2q27x2e`
-  A :t:`field access expression` where the :t:`type` of the :t:`container
-  operand` is :t:`mutable`,
-
-* :dp:`fls_ilaqmj3hc5uv`
-  A :t:`path expression` that resolves to a :t:`mutable static`,
-
-* :dp:`fls_m0gbw9myylv2`
-  A :t:`path expression` that resolves to a :t:`mutable variable` that is not
-  currently borrowed,
-
-* :dp:`fls_dcm3yr3y9y0a`
-  A :t:`temporary` created for a :t:`value expression`.
+:dp:`fls_gq35gqagw35`
+A :t:`place expression` shall not be evaluated in a :t:`value expression
+context` or be bound *by value* in a :t:`pattern` unless it implements
```

Comment:
`by value` should be a reference to the term `by value`. I actually fixed this, so perhaps your sources are out of date.